### ISSUE TITLE
Flaky test

### ISSUE THREAD

**Issue by Dogacel** [line/kotlin-jdsl#610]
For `:hibernate-example:test`
- DeleteExample - [delete all books published after June 2023()]
- SelectExample - [books published between January and June 2023()]

For `:hibernate-reactive-example:test`
- DeleteMutinySessionExample - [delete all books published after June 2023()]
- DeleteMutinyStatelessSessionExample - [delete all books published after June 2023()
- DeleteStageSessionExample - [delete all books published after June 2023()]
- DeleteStageStatelessSessionExample - [delete all books published after June 2023()]
- SelectMutinySessionExample - [books published between January and June 2023()]
- SelectMutinyStatelessSessionExample - [books published between January and June 2023()]
- SelectStageSessionExample - [books published between January and June 2023()]
- SelectStageStatelessSessionExample - [books published between January and June 2023()]

Error:

```
org.opentest4j.AssertionFailedError: 
expected: 
  [Isbn(value=01), Isbn(value=02), Isbn(value=03), Isbn(value=04), Isbn(value=05)]
 but was: 
  [Isbn(value=01),
      Isbn(value=02),
      Isbn(value=03),
      Isbn(value=04),
      Isbn(value=05),
      Isbn(value=06)]
```

Basically it doesn't delete the `Isbn(value=06)` sometimes. I suspect there is a flakiness with timezones.

To reproduce, consider changing your machine's timezone to CST (GMT-6) which is my timezone.

**Comment by shouwn** [comment]
I found the cause. It was because hibernate was changing the timezone of the connection arbitrarily. I added the following 
settings to fix the timezone to the timezone of the running environment. 

```
<property name="hibernate.timezone.default_storage" value="NORMALIZE"/>
```

I pushed it to the develop branch, please check it.

**Comment by shouwn** [comment]
Let me know if you face the same issue. I'll close this issue.
